### PR TITLE
Playlist

### PR DESCRIPTION
**Pull request by krgarima**
##Change Logs:

#Description: Playlist Management

## More details:
1. User can `create playlist` and `delete playlist`
2. `Add videos` to an existing playlist or to the newly created playlist
3. `Delete video` from a playlist
4. `See playlists` in playlist page
5. `See videos` in a playlist
6. Used `Redux` to implement all these features

What not to check?
- Responsiveness
- Using css variables

Screenshots:
av9
av10
av13

### CODE REVIEW

**Review by Asuraking1n — APPROVED** [review]
Working as expected.
Everything looks good
You can merge

File: src/app/store.js [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+import deletePlaylistReducer from "../features/playlist/deletePlaylist-slice";
 
 export const store = configureStore({
   reducer: {
     likes: likesReducer,
     watchLater: watchLaterReducer,
+    playlists: playlistReducer,
+    playlist: playlistReducer,
+    wholePlaylist: playlistReducer,
+    // playlistVideos: playlistReducer,
```
Comment (ranitasahaa1230) [inline src/app/store.js:14]: Remove  this if not needed

File: src/components/ShowPlaylist/ShowPlaylist.js [hunk 2]
```diff
@@ -1,5 +1,62 @@
...
+  const encodedToken = localStorage.getItem("token");
+  let dispatch = useDispatch();
+  let playlists = useSelector((state) => state.playlist.wholePlaylist);
+
+  // let playlistVideos = useSelector((state) => state.playlist.playlistVideos);
+  // console.log(playlistVideos);
+
+  // useEffect(() => {
+  //   dispatch(getPlaylistVideos(videoId.videoId));
+  // }, []);
```
Comment (ranitasahaa1230) [inline src/components/ShowPlaylist/ShowPlaylist.js:21]: Remove all these lines if not needed

**Review by ranitasahaa1230 — APPROVED** [review]
Functionality is working proper
Looks good to merge
Do remove the lines which are not needed

**Review by yogesh- — APPROVED** [review]
- works as expected
- code is readable
okay to merge

**Pull request merged**